spl-ids/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+pub mod atoken;
```

Comment:
I'd prefer to avoid shorthands.
```suggestion
pub mod associated_token_account;
```

Reply:
will change